context/brand-context/springernature/scss/10-settings/buttons.scss

```diff
@@ -20,28 +22,29 @@ $context--button-transition: background-color 0.2s;
  */
 
 $context--button-default: (
-	'color': white,
-	'backgroundColor': color('midnight'),
+	'color': color('white'),
+	'backgroundColor': $context--button-brand-color,
 	'backgroundImage': none,
-	'border': $context--button-border-width-style color('midnight'),
-	'visitedColor': white,
-	'hoverColor': color('midnight'),
-	'hoverBorder': $context--button-border-width-style color('midnight'),
-	'hoverBackgroundColor': white,
+	'border': $context--button-border-width-style $context--button-brand-color,
+	'visitedColor': color('white'),
+	'hoverColor': $context--button-brand-color,
+	'hoverBorder': $context--button-border-width-style color('white'),
```

Comment:
color('white') should be $context--button-brand-color according to previous style